indexer/src/test/java/uk/ac/ebi/uniprot/indexer/uniprotkb/UniProtKBJobIT.java

```diff
@@ -54,18 +57,46 @@ void testUniProtKBIndexingJob() throws Exception {
         BatchStatus status = jobExecution.getStatus();
         assertThat(status, is(BatchStatus.COMPLETED));
 
-        Page<UniProtDocument> response = template
-                .query(SolrCollection.uniprot.name(), new SimpleQuery("*:*"), UniProtDocument.class);
+        Collection<StepExecution> stepExecutions = jobExecution.getStepExecutions();
+        assertThat(stepExecutions, hasSize(2));
+
+        checkUniProtKBIndexingStep(jobExecution, stepExecutions);
+        checkSuggestionIndexingStep(stepExecutions);
+    }
+
+    private void checkSuggestionIndexingStep(Collection<StepExecution> stepExecutions) {
+        StepExecution suggestionIndexingStep = stepExecutions.stream()
+                .filter(step -> step.getStepName().equals(SUGGESTIONS_INDEX_STEP))
+                .collect(Collectors.toList()).get(0);
+
+        assertThat(suggestionIndexingStep.getReadCount(), is(greaterThan(0)));
+
+        int reportedWriteCount = suggestionIndexingStep.getWriteCount();
+        assertThat(reportedWriteCount, is(greaterThan(0)));
+        assertThat(suggestionIndexingStep.getSkipCount(), is(0));
+        assertThat(suggestionIndexingStep.getFailureExceptions(), hasSize(0));
+
+
+        Page<SuggestDocument> response = template
+                .query(SolrCollection.suggest.name(), new SimpleQuery("*:*"), SuggestDocument.class);
         assertThat(response, is(notNullValue()));
-        assertThat(response.getTotalElements(), is(5L));
+        assertThat(response.getTotalElements(), is((long)reportedWriteCount));
```

Comment:
Can you please also check if the document content is correct? We have many stored fields and it can be checked....

Reply:
done as chatted about for accessions